Mathlib/Lean/Meta/CongrTheorems.lean

```diff
@@ -28,17 +28,21 @@ Note that this is slightly abusing `.subsingletonInst` since
 (2) the argument might not even be an instance. -/
 def mkHCongrWithArity' (f : Expr) (numArgs : Nat) : MetaM CongrTheorem := do
   let thm ← mkHCongrWithArity f numArgs
-  process thm thm.type thm.argKinds.toList #[] #[] #[]
+  process thm thm.type thm.argKinds.toList #[] #[] #[] #[]
 where
   /-- Process the congruence theorem by trying to pre-prove arguments using `prove`. -/
```

Comment:
If it's fresh on your mind, could you add a comment here explaining the meaning of `param`, `args`, and `letArgs`?

Reply:
Added